main/updater.js

```diff
@@ -69,6 +71,7 @@ autoUpdater.on('download-progress', progressObj => {
 
 // Update Downloaded
 autoUpdater.on('update-downloaded', info => {
+  Notify({ title: 'Manta Update', body: 'Updates has been downloaded' });
```

Comment:
This needs translation. Please refers to PR #187 for implementation detail of i18next